param/parameterized.py

```diff
@@ -1997,7 +2002,7 @@ def _update_event_type(self_, watcher, event, triggered):
         Returns an updated Event object with the type field set appropriately.
         """
         if triggered:
-            event_type = 'triggered'
+            event_type = 'triggered' if triggered == True else triggered
```

Comment:
The docs should explain what values for `triggered` will be accepted, because otherwise it's very hard to reason about what values event_type might end up with.

Reply:
I.e. is it simply a Boolean? If so, event_type can be 'triggered', 'changed', or False? Seems odd.